modules/seo-tools/jetpack-seo-utils.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * Class containing utility static methods that other SEO tools are relying on.
+ */
+class Jetpack_SEO_Utils {
+	/**
+	 * Site option name used to store front page meta description.
+	 */
+	const FRONT_PAGE_META_OPTION = 'advanced_seo_front_page_description';
+
+	/**
+	 * Old version of option name that was previously used under Free plan.
+	 */
+	const GRANDFATHERED_META_OPTION = 'seo_meta_description';
+
+	/**
+	 * Used to check whether SEO tools are enabled for given site.
+	 *
+	 * @param int $blog_id Optional. Defaults to current blog id if not given.
+	 *
+	 * @return bool True if SEO tools are enabled, false otherwise.
+	 */
+	public static function is_enabled_jetpack_seo( $blog_id = 0 ) {
+		if ( empty( $blog_id ) ) {
+			$blog_id = get_current_blog_id();
```

Comment:
I've added a plan check that is performing API request in 4fb59699c565278f67b80af0a0c86b2f1d1df239. It's just for testing purposes while I work out something better. I'm not so much concerned with page load times, as I'm with every Jetpack site making a request to our API on every page view.